deploy/mod/deploy.py

```diff
@@ -669,6 +685,7 @@ def make_osd_fs(self, osd, osd_num, osd_device, journal_device, mount_list):
         osd_filedir = osd_filename.replace("$id", str(osd_num))
         common.pdsh( user, [osd], 'mkdir -p %s/%s' % (osd_basedir, osd_filedir))
         common.pdsh( user, [osd], 'mount %s -t xfs %s %s/%s' % (mount_opts, osd_device, osd_basedir, osd_filedir))
+        self.make_osd(osd, osd_num, osd_device, journal_device)
```

Comment:
Why put make_osd here? I don't think we will go to make_osd_ceph_disk_prepare when not using ceph-disk.